README.md

```diff
@@ -37,6 +37,50 @@ For an example of how to use this in an overall Modern Data Warehouse solution a
 
 For additional usage information, see [caching pipeline runs](#Caching-pipeline-runs).
 
+For testing storage event triggered pipelines, please use another fixture `adf_storage_event_triggered_pipeline_run`.
+
+```python
+PIPELINE_NAME = "my_blobcreated_event_triggered_pipeline"
+TRIGGER_NAME = "storage_blob_created_event_trigger"
+TRIGGER_FILE_PATH = "event-check/TR_BLOB_CREATED_STORAGE_EVENT.txt"
+CONTAINER_NAME = "event-trigger"
+LOCAL_TRIGGER_FILE_PATH = "./resources/TR_BLOB_CREATED_STORAGE_EVENT.txt"
+
+def test_pipeline_succeeded(adf_storage_event_triggered_pipeline_run):
+    this_run = adf_storage_event_triggered_pipeline_run(PIPELINE_NAME,
+                                                        TRIGGER_NAME,
+                                                        TRIGGER_FILE_PATH,
+                                                        CONTAINER_NAME,
+                                                        LOCAL_TRIGGER_FILE_PATH)
+    
+    assert this_run.status == "Succeeded"
+```
+
+Unlike `adf_pipeline_run` fixture, the `adf_storage_event_triggered_pipeline_run` fixture firstly helps uploading declared storage event trigger file to target container of storage account to trigger the storage-event-triggered pipeline. Then it will block and poll related triggered pipeline run till completion* before returning. Pipeline run completion definition is same as above.
+
+Note that 
+- since pipeline parameters are required to set default values when adding/editing pipeline triggers, therefore there's no *run_inputs* argument for this fixture.
+- since this fixture also helps to upload/delete event trigger file to/from storage account, we need additional storage account related environment variables, more details please refers to below *Environment Variables* section.
+- please provide the target event trigger file as test resources when testing `BlobCreated` event trigger pipelines.
+- `BlobCreated` is the default blob event type, for `BlobDeleted` event trigger pipeline tests, please refer to below sample instead, which just skips the local-trigger-file-path argument and assigns `BlobDeleted` value to *blob_event_type* explicitly, then this fixture will help to delete declared event trigger file from the target container to trigger the specified pipeline run.
+
+
+```python
```

Comment:
Move this up sample code up right after the BlobCreated sample code, but put another bolded title: **"Storage Event: BlobDeleted"**

I think if they are side by side it would be easier to spot the difference :)